src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -581,27 +615,33 @@ blockToMarkdown' opts t@(Table _ blkCapt specs thead tbody tfoot) = do
   return $ nst (tbl $$ caption'') $$ blankline
 blockToMarkdown' opts (BulletList items) = do
   contents <- inList $ mapM (bulletListItemToMarkdown opts) items
-  return $ (if isTightList items then vcat else vsep) contents <> blankline
+  variant <- asks envVariant
+  return $ (if isTightList items || variant == Markua then vcat else vsep) 
+                  contents <> blankline
 blockToMarkdown' opts (OrderedList (start,sty,delim) items) = do
   variant <- asks envVariant
   let start' = if variant == Commonmark || isEnabled Ext_startnum opts
                   then start
                   else 1
   let sty'   = if isEnabled Ext_fancy_lists opts then sty else DefaultStyle
-  let delim' = if isEnabled Ext_fancy_lists opts
-                  then delim
-                  else if variant == Commonmark &&
-                           (delim == OneParen || delim == TwoParens)
-                       then OneParen -- commonmark only supports one paren
-                       else DefaultDelim
+  let delim' | isEnabled Ext_fancy_lists opts =
+               case variant of
+                   Markua -> if delim == TwoParens then OneParen else delim
+                   _ -> delim
+             | variant == Commonmark && --commonmark only supports one paren
+                   (delim == OneParen || delim == TwoParens) = OneParen
+             | otherwise = DefaultDelim
```

Comment:
I wonder if it makes more sense to put the check for Markua outside of this guard (like the check for Commonmark below).  Then we don't need to enable `fancy_lists` extension.  This is connected to the question above, whether it makes sense to have extensions for Markua.

Reply:
The fancy_lists extension is used, because Markua supports the use of "special" enumerators (like roman numbers). Even though in this case it would be more elegant to test for Markua in a different way, this would require to add a special case in every time fancy-lists are possible. To keep the code clean and without unnecessary duplications, we opted for using the option instead as with it the Markdown part is identical to Markua.